### ISSUE TITLE
If a file contains {}, the note is automatically saved as a flashcard

### ISSUE THREAD

**Issue by owlasea** [reuseman/flashcards-obsidian#82]
First off, thanks for the great work! This plugin has been great!

The problem I encountered is that when using the plugin and your file contains "{}" (e.g., {ABC}), you don't even need to add the necessary tag; each line automatically becomes a new card in the default deck.

I believe it's not intentional but a bug. 

The last release without this bug is 1.5.4.

**Comment by reuseman** [comment]
Hey @owlasea, that's the intentional behaviour. Whenever the `{syntax}` it's used, then it's going to be a clozecard. Even for the inline cards, there is no need for the tag. The tag is only used for the normal flashcards.

**Comment by owlasea** [comment]
I see. Thanks for the clarification. Is it possible to make this optional or is there anything I could tweak in the plugin to disable it for my personal use though? I have a bunch of notes with {} in the lines and making it cloze cards means exporting hundreds of new cards each time I save it. Thanks again!

**Comment by wolfresendiz** [comment]
This issue crash directedly with other feature: The latex Support.

for instance:

```
what's a half? #card 
$\frac{1}{2}$
```
adds two cards.

**Comment by katrinleinweber** [comment]
Would it be feasible to ignore `{}` within pre-formatted blocks? I stumbled over the problem right after installing this plugin: _regular expressions_ like ``` `\w{2}` ``` on a note were turned into cards

**Comment by reuseman** [comment]
@katrinleinweber yeah, that's possible. I don't know when I will have time to do it though.

**Comment by reuseman** [comment]
This v1 false-positive behavior is fixed in Flashcards 2.0.0. Plain braces such as `{ABC}` no longer create cards. V2 recognizes `{N:text}`, `==text==`, and native `{{cN::text}}` clozes, while cloze-like text inside code or math remains ordinary content.

Docs: https://github.com/reuseman/flashcards-obsidian/blob/main/docs/wiki.md#cloze
Release: https://github.com/reuseman/flashcards-obsidian/releases/tag/2.0.0
